### PR TITLE
Move Navigable Toolbar tests into Playwright

### PR DESCRIPTION
**Pull request by jeryj**
## What?
Porting over the few navigable toolbar tests from puppeteer into playwright. Some already had coverage in Playwright.

## Why?

I have another PR that will touch some of these, and figured I'd take the time to move these over.

## Testing Instructions

npm run test:e2e:playwright test/e2e/specs/editor/various/navigable-toolbar.spec.js

### CODE REVIEW

File: packages/e2e-tests/specs/editor/various/navigable-toolbar.test.js [hunk 1]
```diff
@@ -1,104 +0,0 @@
...
-
-			// [TEMPORARY]: A new paragraph is not technically a block yet
-			// until starting to type within it.
-			await page.keyboard.type( 'Example' );
-
-			// Upward.
-			await pressKeyWithModifier( 'alt', 'F10' );
-
-			expect( await isInBlockToolbar() ).toBe( true );
-		} );
```
Comment (jeryj) [inline packages/e2e-tests/specs/editor/various/navigable-toolbar.test.js:65]: Already covered by https://github.com/WordPress/gutenberg/blob/eee90667586f38ac1f45d75370357880c3cadadf/test/e2e/specs/editor/various/shortcut-focus-toolbar.spec.js#L12

File: packages/e2e-tests/specs/editor/various/navigable-toolbar.test.js [hunk 2]
```diff
@@ -1,104 +0,0 @@
...
-
-			// Upward.
-			await pressKeyWithModifier( 'alt', 'F10' );
-
-			expect(
-				await page.evaluate( () => {
-					return document.activeElement.getAttribute( 'aria-label' );
-				} )
-			).toBe( 'Show document tools' );
-		} );
```
Comment (jeryj) [inline packages/e2e-tests/specs/editor/various/navigable-toolbar.test.js:102]: Already covered by https://github.com/WordPress/gutenberg/blob/eee90667586f38ac1f45d75370357880c3cadadf/test/e2e/specs/editor/various/shortcut-focus-toolbar.spec.js#L79

**Comment by Mamaduka** [comment]
Hi, @jeryj

It looks like there's already an in-progress PR for migration - #50900. Maybe you can share your feedback there.

**Comment by jeryj** [comment]
Closing in favor of https://github.com/WordPress/gutenberg/pull/50900

**Pull request closed without merging**